src/ripple/app/ledger/Ledger.cpp

```diff
@@ -619,8 +620,9 @@ Ledger::setup(Config const& config)
     {
         ret = false;
     }
-    catch (std::exception const&)
+    catch (std::exception const& ex)
     {
+        JLOG(j_.error()) << "Exception in " << __func__ << ": " << ex.what();
```

Comment:
```__func``` is part of the C++ language since C++11 according to [cppreference.com](https://en.cppreference.com/w/cpp/preprocessor/replace).

Also, it is already used at difference locations in the code base, e.g. [here](https://github.com/XRPLF/rippled/blob/develop/src/ripple/core/impl/JobQueue.cpp#L91).